src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -416,7 +416,7 @@ def wrapper(*args, **kwargs):
 end_partition_invoked = InvokedFlag()
 
 class {_DEFAULT_HANDLER_NAME}(func):
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
Probably we should avoid these two because they are generated code and type hint is not needed.